software/opensign.yml

```
@@ -0,0 +1,16 @@
+name: "OpenSign™"
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted DocuSign alternative."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - MongoDB
+  - Docker
```

See below: Mongodb isnt  platform but does require 3rd party flag.
```suggestion
platforms:
  - Nodejs
  - Docker
```